ecc/blocks/event-info-component/controller.js

```diff
@@ -190,13 +190,41 @@ export function onSubmit(component, props) {
   setPropsPayload(props, eventInfo);
 }
 
+const disableMczEventFields = (component, disabled = true) => {
+  const fieldSelectors = [
+    '#info-field-event-title',
+    '#info-field-event-description', 
+    '#event-info-date-picker',
+    '#time-picker-start-time',
+    '#time-picker-end-time',
+    '#time-zone-select-input',
+    'ampm-picker-start-time',
```

Comment:
Missing #'s? (this line and the following) I don't think component.querySelector(ampm-picker-start-time) will give you anything.